server.py

```diff
@@ -111,7 +112,10 @@ async def healthcheck(request):
     return web.Response(status=200, text="OK")
 
 
-def url_fetcher(url):
+def url_fetcher(url, *args):
```

Comment:
Why are you adding additional positional arguments here? The default url fetcher does not take any additional positional arguments (https://github.com/Kozea/WeasyPrint/blob/master/weasyprint/urls.py#L171). However it has some keyword arguments. But those are not passed when calling the fetcher (https://github.com/Kozea/WeasyPrint/blob/master/weasyprint/urls.py#L246).

Reply:
I'm not sure why, but I've pushed new changes without those positional arguments now!